### ISSUE TITLE
Documentation bug: glPca parallel is FALSE by default, docs say it's TRUE

### ISSUE THREAD

**Issue by plantarum** [thibautjombart/adegenet#298]
The manual for `glPca` says the default value for the `parallel` argument is `TRUE`:

https://github.com/thibautjombart/adegenet/blob/78be588d418f8e5b0a05ebc2880917b1c6581054/man/glPca.Rd#L69-L72

It's actually `FALSE` in the code:

https://github.com/thibautjombart/adegenet/blob/78be588d418f8e5b0a05ebc2880917b1c6581054/R/glFunctions.R#L279-L280

**Comment by zkamvar** [comment]
Good point! We changed the default a while ago because it's better to assume people do not have parallel support but forgot to change the docs.

**Comment by dylanHco** [comment]
Hello:

I think my error might have something to do with this... I am running glPca on jupyter notebooks. I was able to get it to work the first time, but now every time I run it the R kernel dies. I have tried parallel=FALSE and parallel=TRUE... any ideas to get it working on juypter notebooks? 

Thanks!

**Comment by dylanHco** [comment]
I got around this by running: 

`lite.pca <- glPca(lite, nf = 3, parallel=TRUE, n.cores = 10, useC = FALSE)`

and then had to run this for dapc:

`San.dapc <- dapc(lite, n.pca = 3, n.da = 2, parallel=TRUE, n.cores = 10, useC = FALSE)`

-Dylan
